src/reducers/requests/posts.js

```diff
@@ -46,6 +46,17 @@ function getPostThread(state = initialRequestState(), action) {
     );
 }
 
+function getPostThreadWithRetryAttempt(state = 0, action) {
+    switch (action.type) {
+    case PostTypes.GET_POST_THREAD_WITH_RETRY_ATTEMPT:
+        return state + 1;
```

Comment:
@jarredwitt ^

but if I understood correctly is to set the number of retry attempts so far. I'm I right?